src/components/rewards-popup/constants.ts

```diff
@@ -0,0 +1,27 @@
+export const rewardsFaq = [
+  {
+    content:
+      'A product does not exist without the people who use it. By using Zero, you are not doing nothing, you are adding value and we believe you should be rewarded for that.As a web3 platform, Zero rewards are not just cash, they have additional uses that will be added over time, such as; voting, creating your own networks and launching DAOs & NFTs. See Wilder World for an example of what Zero tech can do Your participation in most large centralised social networks results in large profit to the company through advertisement - the monetisation of attention. If you produce something amazing and millions of people see it, millions of people see ads too and that money goes to the company, NOT YOU. We think this needs to change. You are creating the value, so you should share in the revenue',
+    title: 'This seems too good to be true, how can I earn for doing nothing?',
+  },
+  {
+    content:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum bibendum bibendum metus, sit amet auctor erat rutrum ac. Curabitur nec lectus in tellus egestas aliquet non ac lorem. Curabitur id mauris nisi. Ut velit felis, posuere eu gravida quis, varius ullamcorper leo. Sed interdum posuere diam vitae facilisis. Sed commodo consectetur est consequat aliquam. Vestibulum vel tempor nibh, a pellentesque erat. Ut mauris orci, vehicula ut sagittis quis, eleifend vel ante. Maecenas sed lacinia dolor. Vivamus condimentum hendrerit turpis vitae porta. Nunc malesuada massa sit amet sem aliquet, non posuere odio placerat. Curabitur leo ipsum, ultricies sed finibus quis, sodales.',
```

Comment:
Did you check with the design team to see if there's a list of questions/answers?

Reply:
Haha. I just saw your message in the chat. You did indeed ask :)

Reply:
:)